packages/elements/src/utils/third-party-strategies.ts

```diff
@@ -0,0 +1,77 @@
+import type { EnvironmentResource, OAuthProvider, OAuthStrategy, Web3Provider, Web3Strategy } from '@clerk/types';
+import { OAUTH_PROVIDERS, WEB3_PROVIDERS } from '@clerk/types'; // TODO: This import shouldn't be part of @clerk/types
+
+import { fromEntries, iconImageUrl } from './clerk-js';
+
+// c.f. vendor/clerk-js/src/ui/hooks/useEnabledThirdPartyProviders.tsx [Modified]
```

Comment:
Let's put this comment at the top of the file and update the path 👀

Reply:
(Doing so in a follow-up PR, to avoid an additional CI run.)